apps/ledger-live-mobile/src/screens/AddAccounts/03-Accounts.tsx

```diff
@@ -111,10 +111,17 @@ function AddAccountsAccounts({
     inline,
     returnToSwap,
   } = route.params || {};
-  // Find accounts that are (scanned && !existing && !used)
-  const newAccountSchemes = scannedAccounts
-    ?.filter(a1 => !existingAccounts.map(a2 => a2.id).includes(a1.id) && !a1.used)
-    .map(a => a.derivationMode);
+
+  const newAccountSchemes = useMemo(() => {
+    // Find accounts that are (scanned && !existing && !used)
+    const accountSchemes = scannedAccounts
+      ?.filter(a1 => !existingAccounts.map(a2 => a2.id).includes(a1.id) && !a1.used)
```

Comment:
Coming too late but I would've made `existingAccounts.map(a2 => a2.id)` an array outside of the filter function as it's being called for every individual element of `scannedAccounts`. I think we don't scan too many accounts at once but it's an easy perf win